src/main/java/com/coravy/hudson/plugins/github/GithubLinkAction.java

```diff
@@ -30,4 +35,24 @@ public String getUrlName() {
         return projectProperty.getProjectUrl().baseUrl();
     }
 
+    @SuppressWarnings({"rawtypes"})
+    @Extension
+    public static class Factory extends TransientActionFactory<Job> {
```

Comment:
Please name it `GithubLinkActionFactory` for better stacks in case of errors and debug